bitsandbytes/autograd/_functions.py

```diff
@@ -490,7 +497,9 @@ class MatMul4Bit(torch.autograd.Function):
     # backward is mostly the same, but adds one extra clause (see "elif state.CxB is not None")
 
     @staticmethod
-    def forward(ctx, A, B, out=None, bias=None, quant_state: Optional[F.QuantState] = None):
+    def forward(
+        ctx, A: torch.Tensor, B: torch.Tensor, out=None, bias=None, quant_state: Optional[F.QuantState] = None
```

Comment:
Add the trailing comma so Ruff will spread this out onto multiple lines for ease of reading? (Will repeat this as "trailing comma?" throughout this review)
```suggestion
        ctx, A: torch.Tensor, B: torch.Tensor, out=None, bias=None, quant_state: Optional[F.QuantState] = None,
```